examples/generative/stylegan.py

```diff
@@ -74,7 +98,7 @@ def resize_image(res, sample):
 def create_dataloader(res):
     batch_size = batch_sizes[log2(res)]
     dl = ds_train.map(partial(resize_image, res), num_parallel_calls=tf.data.AUTOTUNE)
-    dl = dl.shuffle(200).batch(batch_size, drop_remainder=True).prefetch(1).repeat()
+    #dl = dl.shuffle(200).batch(batch_size, drop_remainder=True).prefetch(1).repeat()
```

Comment:
This should be removed, right?

Reply:
Actually, I was having trouble with this code to support dynamic batch sizes and just found the workaround: adding an `unbatch()` call. 

It seems the KPL `image_dataset_from_directory()` method returns `BatchDataset` types and applying a `batch()` call to them is... unwise.